rest-server/engines/src/main/scala/com/actionml/engines/ur/UREngine.scala

```diff
@@ -144,6 +144,7 @@ class UREngine extends Engine with JsonSupport {
     logger.info(s"Dropping persisted data for Engine-id: $engineId")
     dataset.destroy()
     algo.destroy()
+    JobManager.getActiveJobDescriptions(engineId).foreach(d => JobManager.cancelJob(engineId, d.jobId))
```

Comment:
@pferrel is it ok to remove all jobs of the engine on engine.destroy?

Reply:
Yes